setup.cfg

```diff
@@ -43,7 +43,7 @@ single-quotes = true
 
 [tool:pytest]
 minversion = 5.4
-addopts = -p no:doctest --xdoctest --xdoctest-style=google
+addopts = -p no:doctest --xdoctest --xdoctest-style=google -r fEs
```

Comment:
@mmulich Please explain these additional arguments

Reply:
This setting `-r fEs` includes fails, errors and skips at the end of
the execution. The skip was the important one for me. I like to know
why a test was skipped. The "reason" information is now provided in the output.